packages/cli/src/create/helpers/create-app.ts

```diff
@@ -43,7 +45,9 @@ export async function createApp({
       process.exit(1);
     }
   } else if (framework) {
-    frameworkPath = `${framework}-${language || "javascript"}-starter`;
+    frameworkPath = `${framework}-${language || "javascript"}-${
+      chain && "solana-"
+    }starter`;
```

Comment:
we should *explicitly* check for chain === solana in this case, because I would assume that this would be the same thing for other chains down the line then (also again ,`evm` should have it's own "chain" value